workers/generic-worker/gwconfig/config.go

```diff
@@ -29,7 +28,7 @@ type (
 
 	PublicConfig struct {
 		PublicEngineConfig
-		AuthBaseURL                    string                 `json:"authBaseURL"`
+		AuthRootURL                    string                 `json:"authRootURL"`
```

Comment:
I think this will require a README update as well, and it would be good to mention that the *RootURL configs are only for use in testing and development.